lib/tools/adb-commands.js

```diff
@@ -1071,7 +1071,7 @@ methods.broadcast = async function (intent) {
  * Kill Android instruments if they are currently running.
  */
 methods.endAndroidCoverage = async function () {
-  if (this.instrumentProc) {
+  if (this.instrumentProc && this.instrumentProc.isRunning) {
```

Comment:
is it a property?

Reply:
It's a getter.

Reply:
https://github.com/appium/node-teen_process/blob/master/lib/teen_process.js#L131-L134